### PR TITLE
ENH: GCS RSSI Support

### PR DESCRIPTION
**Pull request by BrainFPV**
Adds RSSI to the input configuration in the GCS and also adds an additional column showing the current value for each channel.

rssi

### CODE REVIEW

**Comment by peabody124** [comment]
Nice. Stylistically, do you think you could make the 'Current' column a label rather than edit field.

**Comment by DTFUHF** [comment]
Nice.

**Comment by BrainFPV** [comment]
@peabody124 yes, that looks better :)

rssi1

**Comment by mlyle** [comment]
Beautiful, a hint as to current value is going to be so useful during setup.

File: ground/gcs/src/plugins/config/inputchannelform.h [hunk 1]
```diff
@@ -12,7 +12,7 @@ class inputChannelForm : public ConfigTaskWidget
     Q_OBJECT
 
 public:
-    explicit inputChannelForm(QWidget *parent = 0,bool showlegend=false);
+    explicit inputChannelForm(QWidget *parent = 0,bool showlegend=false, bool showSlider=true);
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/inputchannelform.h]: Spacing on args (I know there's tons of violations here, but if you're touching the line it should probably be fixed or at least consistent).

File: ground/gcs/src/plugins/config/configinputwidget.cpp [hunk 2]
```diff
@@ -81,7 +81,7 @@ ConfigInputWidget::ConfigInputWidget(QWidget *parent) : ConfigTaskWidget(parent)
     foreach (QString name, manualSettingsObj->getField("ChannelNumber")->getElementNames())
     {
         Q_ASSERT(index < ManualControlSettings::CHANNELGROUPS_NUMELEM);
-        inputChannelForm * inpForm=new inputChannelForm(this,index==0);
+        inputChannelForm * inpForm=new inputChannelForm(this,index==0,true);
```
Comment (mlyle) [inline ground/gcs/src/plugins/config/configinputwidget.cpp]: Spacing on assignment, possibly on args.

**Comment by mlyle** [comment]
Looks great.

**Comment by BrainFPV** [comment]
@mlyle thanks! I adjusted the spacing for consistency.

**Comment by elafargue** [comment]
I like it! Nice enhancement

**Pull request merged**